clang/runtime/dpct-rt/include/sparse_utils.hpp

```diff
@@ -262,6 +262,469 @@ void optimize_csrsv(sycl::queue &queue, oneapi::mkl::transpose trans,
 }
 #endif
 
+/// Structure for describe a dense vector
+struct dense_vector_desc {
+  dense_vector_desc(std::int64_t ele_num_data, void *value_data,
+                    library_data_t value_type_data)
+      : ele_num(ele_num_data), value(value_data), value_type(value_type_data) {}
+  void get_desc(std::int64_t *ele_num_data, const void **value_data,
+                library_data_t *value_type_data) {
+    *ele_num_data = ele_num;
+    *value_data = value;
+    *value_type_data = value_type;
+  }
+  void get_desc(std::int64_t *ele_num_data, void **value_data,
+                library_data_t *value_type_data) {
+    get_desc(ele_num_data, const_cast<const void **>(value_data),
+             value_type_data);
+  }
+  std::int64_t ele_num;
+  void *value;
+  library_data_t value_type;
+};
+
+/// Structure for describe a dense matrix
+struct dense_matrix_desc {
```

Comment:
Let's unify the style of class/struct for dense_matrix_desc and sparse_matrix_desc

Reply:
done